### ISSUE TITLE
Amoled mode still has colored statusbar

### ISSUE THREAD

**Issue by bin101** [ssaqua/ProximityService#8]
I think that's because my device (onplus 6) has a notch.

Here a screenshot:
![screenshot_20181204-210923](https://user-images.githubusercontent.com/12427722/49470108-6fddc880-f809-11e8-8f17-37603644ee7c.jpg)

**Comment by ssaqua** [comment]
I'm guessing you are running Android Pie. Could you check if the attached build would solve this for you?

(Or you could build the latest debug build off the display-cutout branch if you like :slightly_smiling_face: )

[app-debug.apk.zip](https://github.com/ssaqua/ProximityService/files/2648075/app-debug.apk.zip)

**Comment by bin101** [comment]
Yes I use Android 9 and your provided debug build worked fine :)
